week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -24,9 +41,12 @@ The function must:
 - return a new array with the item removed
 - remove the item at the specified index
 */
+  // let deleted = arr.splice (2, 1)
 
 function remove(arr, index) {
-  return; // complete this statement
+let newArray = arr.slice()
+newArray.splice(index, 1)
+  return newArray; 
```

Comment:
Nice! Great work on making sure you copy the array!

One small thing, be careful about the indentation here. Good indentation makes code a lot easier to read and understand.